source/camera/basic/capture_hdr.py

```diff
@@ -6,12 +6,15 @@ def _main():
     app = zivid.Application()
     camera = app.connect_camera()
 
-    settings_list = [camera.settings for _ in range(3)]
-    settings_list[0].iris = 14
-    settings_list[1].iris = 21
-    settings_list[2].iris = 35
+    settings = zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(aperture=11.31,),
+            zivid.Settings.Acquisition(aperture=5.66,),
+            zivid.Settings.Acquisition(aperture=2.83,),
+        ],
+    )
```

Comment:
Consider changing to this if you want something more compact:
```
settings = zivid.Settings(
    acquisitions=[zivid.Settings.Acquisition(aperture=fnum) for fnum in (11.31, 5.66, 2.83)])
```

Reply:
done